### ISSUE TITLE
Error updating from local files: Unexpected character '7' found when expecting key.

### ISSUE THREAD

**Issue by Incarceron** [rallion/depressurizer#55]
I'm getting this error when running Depressurizer, not sure what the cause is. I searched and it seems a few others have had this issue as well.

![e2d0ddc6a46f6e56e048a12f6b742fec](https://cloud.githubusercontent.com/assets/2171367/13029843/7136c4a6-d265-11e5-9e07-68e1be592ec0.png)

**Comment by Theo47** [comment]
I'd suggest trying one of the versions from here: https://github.com/Theo47/depressurizer/releases

**Comment by smudge** [comment]
My fix was merged into the 'dev' branch, but this repo seems kind of dead lately. As @Theo47 stated, his builds should include those changes. Might be nice for @rallion to add @Theo47 as a collaborator, so that the work can happen on the main repo instead of on a fork. (Just a thought.)

**Comment by Theo47** [comment]
Yeah your fix made it into the dev branch. Just the dev branch never made it into the master branch.
Noticed that last time i was answering this issue (https://github.com/rallion/depressurizer/issues/47).

Now rallion hasn't been here for over a year so that doesn't seem likely.

**Comment by MonthOLDpickle** [comment]
> Yeah your fix made it into the dev branch. Just the dev branch never made it into the master branch.
> Noticed that last time i was answering this issue (#47).
> 
> Now rallion hasn't been here for over a year so that doesn't seem likely.

So I am getting this error today, how do I find the dev branch?

**Comment by smudge** [comment]
@MonthOldPickle try one of the releases here:

https://github.com/Theo47/depressurizer/releases

**Comment by Zhuinden** [comment]
Fixed in https://github.com/Theo47/depressurizer/issues/64#issuecomment-257153102
